src/new-components/avatar/avatar.tsx

```diff
@@ -52,10 +60,16 @@ function Avatar({ user, avatarUrl, size = 'l', className, colorList = AVATAR_COL
               backgroundColor: colorList[emailToIndex(user.email, colorList.length)],
           }
 
+    const sizeClassName = getClassNames(styles, 'size', avatarSize)
+
     return (
-        <div className={avatarClass} style={style}>
+        <Box
+            className={[className, styles.avatar, sizeClassName, exceptionallySetClassName]}
+            style={style}
+            data-testid="avatar"
```

Comment:
I recommend we do not include in the final component something that's there only to support the internal tests in Reactist.

I recommend that you forward any extra props received by the avatar to the box. You may need some TypeScript tweaking, in which case let me know if you need help. But maybe you can get away with merely adding the `...props` to the Avatar props deconstruction and then forwarding that as `<Box myPropsFirst {...props}>`

Then you pass this `data-testid` in the test only.